_candidates_ja/DevonStewart_2.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Devon Stewart
+title: "より良いFuture[T]の作り方"
+length: 15
+audience: Beginner
+language: English
+twitter: blast_hardchese
+github: blast-hardcheese
+organization: Twilio
+tags:
+  - Functional Programming
+  - Best Practices
+  - Intro to Scala
+  - Software Design and Architecture
+---
+この講演では、ScalaのFuture [T]を使用して、処理の失敗を制御したりやブロックしてしまう処理を避ける抽象化を理解する方法について説明します。 
```

Comment:
原文: In this talk, the speaker will provide insight into how to work with Scala's Future[T] abstraction to avoid blocking and handle failure cases. 
訳案: このトークでは、Scala の Future[T] が提供する抽象化を用いてブロック処理を回避したり、失敗ケースをうまく処理できるのかを解説します。